frontend/src/components/Settings/ProfileInfoSetting.js

```diff
@@ -0,0 +1,118 @@
+// Importing necessary modules
+import React, { useEffect, useState } from "react";
+import {
```

Comment:
Could probably refactor these imports in the future, database query is unlikely to be needed on this page.